pipeline-examples/unstash-different-dir/unstashDifferentDir.groovy

```diff
@@ -27,9 +27,9 @@ node('second-node') {
     }
 
     // Look, no output directory under the root!
-    // pwd() outputs the current directory Pipeline is running in.
-    sh "ls -la ${pwd()}"
+    // PWD outputs the current directory Pipeline is running in.
```

Comment:
Kind of, but not really? The result is the same, assuming you use a shell that supports it, but this won't demonstrate the `pwd()` Pipeline feature. The phrasing seems off for the case where we just use a shell built-in feature.

(Granted, both the existing Pipeline and this one are really artificial, neither `$PWD` nor `pwd()` are actually needed.)